less/mixins.less

```diff
@@ -1 +1,52 @@
-// Mixins Here
\ No newline at end of file
+// Mixins Here
+// Flex mixin
+.flex(
+@flow: row nowrap;
+@justify: flex-start;
+@align-items: stretch;
+@align-content: stretch
+) {
+  display: flex;
+  flex-flow: @flow;
+  justify-content: @justify;
+  align-items: @align-items;
+  align-content: @align-content;
+}
+
+// Column mixin
+.col(@flow: column nowrap; @justify: flex-start; @align: stretch) {
+  .flex(@flow; @justify; @align);
+}
+// Row mixin
+.row(@flow: row nowrap; @justify: flex-start; @align: stretch) {
+  .flex(@flow; @justify; @align);
+}
+
+// Button Mixin
+.btn(@bg-color: black; @color: white) {
+  background-color: @bg-color;
+  border-radius: 0.5rem;
+  border: 1px solid @bg-color;
+  color: @color;
+  cursor: pointer;
+  font-size: 1.9rem;
+  font-weight: bold;
+  padding: 0.75rem 0;
+  text-align: center;
+  transition: color 0.15s ease-in, background-color 0.15s ease-in;
+
+  &:hover {
+    background-color: @color;
+    color: @bg-color;
+    transition: color 0.15s ease-in, background-color 0.15s ease-in;
+  }
+}
+
+// Transition mixin
+.lighten-on-hover(@color) {
+  transition: color 0.15s ease-in;
+  &:hover {
+    color: lighten(@color, 30%);
+    transition: color 0.15s ease-in;
+  }
+}
```

Comment:
- Navigation Styles: Use the navigation.less file for styling. - DONE!

 - Main Content Styles: Use the home-page.less file for styling - DONE!

 - LESS Mixins: Create and use 2 different mixins to aid your styling. Use the mixins.less file for your mixins - DONE!

 LESS Parametric Mixin: create a parametric mixin that is used to create the sign up button styles. - DONE!

 - Use at least 2 parameters to create your button - DONE!

 - Create a hover state that swaps the background color and font color of the base button styles. - DONE!